### PR TITLE
Polished wanderer license descriptions

### PR DESCRIPTION
**Pull request by Onion-S**

### CODE REVIEW

File: data/wanderer outfits.txt [hunk 1]
```diff
@@ -555,14 +555,14 @@ outfit "Type 4 Radiant Steering"
 outfit "Wanderer Outfits License"
 	category "Special"
 	thumbnail "outfit/wanderer outfits license"
-	description "Spoke with a Wanderer ambassador named Iktat Rek using the translation device and received permission to purchase some of their technology, but not their ships."
+	description "After speaking to a Wanderer ambassador named Iktat Rek using a translation device obtained from the Hai, you received permission to purchase the Wanderer's outfits, but not their ships."
```
Comment (tehhowch) [inline data/wanderer outfits.txt]: I think that should be a plural possessive, `Wanderers'`

Comment (Onion-S) [reply data/wanderer outfits.txt]: This is true

**Comment by Amazinite** [comment]
Note that the Wanderer license descriptions are currently intentionally copied and pasted from the log entry that you get upon receiving the license. The same is true for the Coalition license.

File: data/wanderer outfits.txt [hunk 2]
```diff
@@ -555,14 +555,14 @@ outfit "Type 4 Radiant Steering"
...
 	category "Special"
 	thumbnail "outfit/wanderer license"
-	description "Managed to broker a temporary ceasefire between the Unfettered Hai and the Wanderers. Was granted access to purchase the non-military Wanderer ships, as a reward."
+	description "You managed to broker a temporary ceasefire between the Unfettered Hai and the Wanderers. As a reward, you were granted access to Wanderer civilian ships."
 
 outfit "Wanderer Military License"
 	category "Special"
 	thumbnail "outfit/wanderer military license"
-	description "Received permission to purchase warships from the Wanderers, including a new one called the Tempest."
+	description "The Wanderers gave you permission to purchase warships, including the newly-developed Tempest."
```
Comment (Bladewood) [inline data/wanderer outfits.txt]: -ly adverbs are never hyphenated.
```suggestion
	description "The Wanderers gave you permission to purchase warships, including the newly developed Tempest."
```

**Comment by MessyMix** [comment]
You didn't have to close this; it's a good idea! Just because it was intentionally done this way, doesn't mean there isn't room for it to be expanded upon.

I'd say if you want to revise these licenses, start from scratch (i.e. not based off the current log entry) and take it from there. I suspect the current descriptions are just placeholders for more thorough ones in the future, anyways.

**Pull request closed without merging**